src/components/EvSections/about-page-sections/LandingText.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Paragraph } from "components/EvComponents/Typography";
+import { Box , Grid } from "@mui/material";
+import {  SectionTitle } from "components/EvComponents/StyledTypography";
+
+
+
+const LandingText = (props) => {
+  const  {p1 , p2 , p3 }  = props.section;
+  return (
+    
+    <Box sx={{ mt: "10px" , py :5 , px : 6  }}>
+        <SectionTitle>EVENT OVERVIEW</SectionTitle>
+     <Grid container spacing={3} sx = {{pt : 5}}>
+        <Grid item xs={12} sm = {12} md = {12}>
+            <Box sx={{color : "grey.700"}}>
```

Comment:
remove the color. If we have to change it later we will do from the typography component so we have the same color in all paragraphs.

Reply:
Resolved